CONTRIBUTING.md

```diff
@@ -19,23 +19,31 @@ again.
 
 ### General Information
 
- - One PR for one feature or change request. Prefer multiple smaller PRs to one gigantic one. Multiple commits during development are OK; see next point.
+Stratum follows [Google's Engineering Practices](https://google.github.io/eng-practices/review/developer/). Use this document as a guide when submitting code.
+
+Some additional points:
+ 
+ - Submit you changes early and often. GitHub has (Draft PRs)[https://github.blog/2019-02-14-introducing-draft-pull-requests/] that allow you to share your code with others during development. Input and corrections early in the process prevent huge changes later.
  
  - Stratum uses a [squash and rebase](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) model. You do **not** have to do this by hand! GitHub will guide you through it, if possible.
  
- - Describe what your PR solves and why. Consider opening a separate issue describing the technical details there and [link it to the PR](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords). This keeps code review and design discussions clean.
+ - Consider opening a separate issue describing the technical details there and [link it to the PR](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords). This keeps code review and design discussions clean.
 
 ### Steps to Follow
 
-1. Make sure your code passes unit tests. Create new ones for new code and **add them to the [build-targets.txt](.circleci/build-targets.txt) and [test-targets.txt](.circleci/test-targets.txt) files** in the same PR, so CI can pick them up! You can also run them locally with bazel: `xargs -a .circleci/test-targets.txt bazel test`
+1. Fork Stratum into your namespace via the fork button on GitHub.
+
+2. Make your code changes.
+
+3. Pass all unit tests locally. Create new tests for new code and **add the targets to the [build-targets.txt](.circleci/build-targets.txt) and [test-targets.txt](.circleci/test-targets.txt) files** in the same PR, so CI can pick them up! To run all currently enabled tests execute this in the Stratum root directory: `xargs -a .circleci/test-targets.txt bazel test`
```

Comment:
Maybe `Execute the following command in the Stratum root directory to run all currently enabled tests` ?